Unity-Technologies/Graphics
# Do not serialize UI properties on VolumeParameters

---
### Purpose of this PR
https://fogbugz.unity3d.com/f/cases/1312464/

---
### Testing status
- Check the problem of the bug is solved
- Check other parameters
- Change the max to a high number, change the value to that number, decrease the max.

---
### Comments to reviewers
The main problem is that the properties used to modifiy the UI values are serialized, this is correct, but the range values are only set on the constructor,( something that is specified on the Volume Component), therefore they do not need to be serialized.

Just adding the attribute makes them to recover on whatever is on the constructor.
